iOS/G3MiOSSDK/Commons/Basic/ShortBufferElevationData.cpp

```diff
@@ -37,10 +37,49 @@ _buffer(buffer)
       break;
     }
   }
+    
+
+  _max = IMathUtils::instance()->minInt16();
+  _min = IMathUtils::instance()->maxInt16();
+  _children = IMathUtils::instance()->minInt16();
+  _similarity = IMathUtils::instance()->minInt16();
+}
+
+ShortBufferElevationData::ShortBufferElevationData(const Sector& sector,
+                                                   const Vector2I& extent,
+                                                   const Sector& realSector,
+                                                   const Vector2I& realExtent,
+                                                   short* buffer,
+                                                   int bufferSize,
+                                                   double deltaHeight,
+                                                   short max,
+                                                   short min,
+                                                   short children,
+                                                   short similarity) :
+BufferElevationData(sector, extent, realSector, realExtent, bufferSize, deltaHeight),
```

Comment:
Not sure if 'children' (btw, it should be named numberOfChildren or something like that) and 'similarity' should be mandatory for all the ShortBufferElevationData instances.

Probably the same idea applies to delta, max and min.